### ISSUE TITLE
Is notification over MQTT possible?

### ISSUE THREAD

**Issue by bugs181** [gugutab/TvOverlay#51]
Looked everywhere in this repo for examples of how to send a notification via MQTT, with no luck. Is MQTT currently only used for Home Assistant settings and not used for sending notifications? 

Documentation states: 

> If you have a MQTT server running, you can connect your TvOverlay device to it to control its main functions.

I'd argue displaying a notification over MQTT IS a main function. Why tout MQTT at all if it's not possible?

I'd prefer to avoid REST API if at all possible. The HTTP overhead is fairly large on the IoT network compared to MQTT. The less traffic over WiFi the better.

If this was implemented, I'd gladly pay for premium.

**Comment by lazmo88** [comment]
You should be able to build automation that relays MQTT data to notify service. That said, it is definitely doable IMHO.

**Comment by bstromski** [comment]
Agreed, I have multiple devices I want to send notifications to if they’re online. MQTT feels like the right way to do this. Is there any way to accomplish this?
